data.json

```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "name": "DigitalOcean",
```

Comment:
This item has to be moved to be ordered alphabetically. Thanks

Reply:
@vikaspotluri123 Let's add it to the CI tests?

Reply:
Created an issue for it: https://github.com/swapagarwal/swag-for-dev/issues/334